about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="./styles/main.css"> 
+  <link rel="stylesheet" type="text/css" href="./styles/normalize.main.css">
+  <title>About</title>
+</head>
+
+<!-- Navigation Station and Contact/Social Media Links  -->
+<div class="navbar">
```

Comment:
HTML content should be in the `body` section.  

Also instead of using a `div` element consider using a `header` element here, it provides more semantic meaning to the content. 

Good use of an unordered list for the menu.